pkgs/by-name/cs/csv-tui/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "csv-tui";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "nathangavin";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-IRXLwZ2FHcCDmDVJ0xnV/4q+X2AFXPX/+Ph4Xxo3DyM=";
+  };
+
+  cargoHash = "sha256-wgeVcX0zSXffAuvKw2eKXC846WlC8F9UGMoxP3IXoLE=";
+
+  meta = with lib; {
+    description = "A terminal based csv editor which is designed to be not a ram hog like standard csv editors, but more useful than other text editors";
+    homepage = "https://github.com/nathangavin/csv-tui";
+    license = licenses.unfree;
```

Comment:
Ask upstream to license this software so we can redistribute it. See: https://github.com/NixOS/nixpkgs/tree/master/pkgs#quick-start-to-adding-a-package

Reply:
Agh, sorry I missed too many small details when checking the derivation.